### ISSUE TITLE
In case one key is not okay fsck gives up

### ISSUE THREAD

**Issue by DEvil0000** [gopasspw/gopass#2640]
### Summary
In case one key is not okay fsck gives up (in my case it was not an E type key as recipient)

### Steps To Reproduce
* add a key to recipients without E type
* add some others as well
* re-encrypt on add recipient fails
* fsck  --decrypt gives up re-encrypting after seeing the "bad" key

### Expected behavior
re-encrypt for all vaild keys

### Environment
- OS: fedora 38
- OS version: 6.3.13?
- gopass Version: 1.15.4?
- Installation method: package from github releases

**Comment by dominikschulz** [comment]
I don't think this is what we want. If we silently ignore an invalid key that might cause some users to loose access to their keys.

I'd rather expect the user to remove the invalid key and then re-encrypting again.

Do you think there is anything we can do to make this easier?

**Comment by DEvil0000** [comment]
If the error would be more clear and point to the key to remove it would already help. Maybe providing the command needed and some human readable explanation why the key is invalid would be nice.
I however think ignoring the key, warn about it and continue with the fsck/re-encrypt is the correct behavior.
I can imagine use cases where keys may not be added or removed by everyone and only some users but fsck should be possible all the time.

**Comment by dominikschulz** [comment]
I was looking into this a little bit, but it's not completely trivial to fix. Right now we're handling recipients mostly as a black box.

Could anyone provide a step-by-step list of commands to reproduce this, i.e. to create an unuseable GPG key and set up a gopass repo in a way that would trigger this failure mode?
